src/common/helpers/poller.js

```diff
@@ -0,0 +1,272 @@
+import moment from 'moment';
+
```

Comment:
`src/common` is meant for code shared between server and client app, which `poller.js` isn't.

I think it would be better moved to `src/server/` or somewhere around there - as it seems it imports quite a bunch of modules from server anyway.